app/src/main/res/values/dimens.xml

```diff
@@ -6,7 +6,8 @@
 
   <integer name="fade_in_duration_ms">1000</integer>
   <integer name="fade_out_duration_ms">1000</integer>
-  <integer name="profile_chooser_span_count">4</integer>
+  <integer name="profile_chooser_span_count">2</integer>
```

Comment:
Transfer all integers to `integer.xml` file for this PR and previous PRs too.

Reply:
Ok, I'll do it now

Reply:
Done